index.js

```diff
@@ -51,6 +93,23 @@ const createElement = (tagType, attrObj, parentElement) => {
 
     return newElement
 }
+
+const createGlobalWrapper = () => {
```

Comment:
{} are redundant

Reply:
Removed the whole function, since became useless